### ISSUE TITLE
Using Barbra as a knowledge repository

### ISSUE THREAD

**Issue by killah-t-cell** [nodejs/mentorship#140]
As discussed with A.J.,

**I proposed that the Node.JS Mentorship use Barbra as a knowledge repository to supplement its mission of empowering and teaching the Node community.**

Barbra is a platform to store and organise knowledge like your discussion videos, and articles on Node.JS that you guys find particularly useful. It is completely free and it always will be.

https://barbra.io/collection/a19cf626-717a-4b09-b3b0-78a433beed23 this is our current collection of resources on Node.JS which you guys can take over management of if you want. 

As I already mentioned to A.J., we are _not only_ willing but excited to consider making changes to the platform that you guys identify as useful. And you have and will always have complete creative control over what content goes in there. 

In summary, the main benefits of Barbra for you guys are:

- you can store and organize any content in one place.
- the community can follow your collection and get email notifications every time you recommend an article or you guys post a new mentorship video.
- users can upvote and comment on the content you posted so you guys can have conversations about what is helping the community.
- direct contact with the developing team (me and @bitphinix) to make the platform yours.

I am happy to talk more about it with you guys and answer any questions you have regarding the platform. I believe it is a great supplement to your initiative, and I glad to be able to help it in any way.

**Comment by detrohutt** [comment]
@gabrielbirnbaum we discussed this in today's meeting and decided that due to the fact that we're still in the beta phases of the initiative right now, it's in our best interest to keep things as simple as possible for now and not introduce any new variables. Having said that, once we get to the point where the initiative has stabilized and we have more content to share, we can revisit this idea.

**Comment by killah-t-cell** [comment]
@detrohutt It sounds great, you guys. Let's talk about it in the future! 

All the best.
